packages/core/src/docdb/commands/rebootInstance.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+import * as localizedText from '../../shared/localizedText'
+import { getLogger, ToolkitError } from '../../shared'
+import { CancellationError } from '../../shared/utilities/timeoutUtils'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { showConfirmationMessage, showViewLogsMessage } from '../../shared/utilities/messages'
+import { telemetry } from '../../shared/telemetry'
+import { DBInstanceNode } from '../explorer/dbInstanceNode'
+
+/**
+ * Reboots a DocumentDB instance.
+ * Refreshes the parent node.
+ */
+export async function rebootInstance(node: DBInstanceNode) {
+    getLogger().debug('docdb: RebootInstance called for: %O', node)
+
+    await telemetry.docdb_rebootInstance.run(async () => {
+        if (!node) {
```

Comment:
Can you move these into a function thats something like:
```
function assertNode(node: DBResourceNode) {
    if (!node) {
        throw new ToolkitError('No node specified')
    }

    if (!node.isAvailable) {
        void vscode.window.showErrorMessage(
            localize('AWS.docdb.deleteInstance.instanceStopped', 'Instance must be running')
        )
        throw new ToolkitError('Instance not available', { cancelled: true, code: 'docdbInstanceNotAvailable' })
    }
}
```

the exact same code is used like 5+ times

Reply:
refactored into function